setup.py

```diff
@@ -27,50 +27,18 @@ def c_includes(parent, depth=1):
     return includes
 
 
-PACKAGENAME = 'stsci.stimage'
 SOURCES = c_sources('src')
 INCLUDES = c_includes('include') + c_includes('src') + [numpy_includes()]
 
 
 setup(
-    name=PACKAGENAME,
     use_scm_version=True,
```

Comment:
Is `True` just works? Or we need to be more explicit?
```suggestion
    use_scm_version=use_scm_version={'write_to': 'stsci.stimage/version.py'},
```

Reply:
We're not actually writing out a `version.py` file to contain the version info. If `true`, it just writes it to the `package.egg-info/PKG-INFO` in the installed version.  This is how we do it in `jwst`.  Adding the `write_to` part just makes the installed editable install more poke-around-able.  =)

Reply:
Is there any advantage in writing it out to a version.py file?  Faster import time?  Something else?  This is how `stsci.stimage.__version__` is currently implemented:

https://github.com/spacetelescope/stsci.stimage/blob/222d47ba92955975ee661061f5a4481199715531/stsci/stimage/__init__.py#L31-L38

Reply:
Doesn't that make setuptools a runtime requirement as well?

See https://github.com/astropy/astropy/blob/23404eeb2caeea315cea89b6cb8b9b2fee4f13be/astropy/__init__.py#L13

Reply:
Hmm, yes I believe so.  I like the way astropy does it, and most of our other projects that were not early adaptors of `setuptools-scm` write it out as well.  I'll switch it over.